specs/librdkafka.spec

```diff
@@ -122,14 +134,10 @@ rm -rf %{buildroot}
 %files devel
 %defattr(-,root,root,-)
 %{_includedir}/%{name}/*
-%{_libdir}/%{name}.a
```

Comment:
I think it is not a good idea to remove static libraries from the package. It is better to move them to subpackage.